addons/website/views/snippets/s_timeline.xml

```diff
@@ -2,51 +2,60 @@
 <odoo>
 
 <template name="Timeline" id="s_timeline">
-    <section class="s_timeline pt24 pb48">
-        <div class="container s_timeline_line">
-            <div class="s_timeline_row d-block d-md-flex flex-row" data-name="Row">
-                <div class="s_timeline_date"><b>2019</b></div>
-                <div class="s_timeline_content d-flex">
-                    <div class="s_timeline_card s_card card bg-white w-100" data-name="Card" data-snippet="s_card">
-                        <div class="card-body">
-                            <h5 class="card-title">Your title</h5>
-                            <p class="card-text">A timeline is a graphical representation on which important events are marked.</p>
-                        </div>
-                    </div>
-                    <i class="fa fa-1x fa-child bg-o-color-2 rounded-circle s_timeline_icon"/>
+    <section class="s_timeline pt48 pb48" data-vcss="001">
+        <div class="o_container_small">
+            <div class="row">
+                <div class="col-lg-12 pb24 text-center" data-name="Heading">
+                    <h2 class="h3-fs">Latest news</h2>
+                    <p class="lead">Highlight your history, showcase growth and key milestones.</p>
                 </div>
-                <div class="s_timeline_content"/>
             </div>
-            <div class="s_timeline_row d-block d-md-flex flex-row" data-name="Row">
-                <div class="s_timeline_date"><b>2018</b></div>
-                <div class="s_timeline_content d-flex">
-                    <div class="s_timeline_card s_card card bg-white w-100" data-name="Card" data-snippet="s_card">
-                        <div class="card-body">
-                            <p class="card-text">You can edit, duplicate...</p>
+            <div class="position-relative pt-3">
+                <div class="s_timeline_row position-relative d-flex gap-5 align-items-start pb-4" data-name="Milestone">
```

Comment:
This class is necessary to prevent the empty `. s_timeline_content` to be editable.
Without this, the empty `s_timeline_content` takes a full height and you can click on it to edit its content (even when we define `o_not_editable` + `contenteditable="false"`, do you know why btw ? 🤔 ) :

<img width="1478" alt="Capture d’écran 2024-07-30 à 15 42 25" src="https://github.com/user-attachments/assets/be93d027-ea1a-4e83-b517-a27d335cf622">


Defining this class will reduce the height of this element to 0, and it will be neither clickable nor editable.

Another solution would be to define `pe-none` on empty `s_timeline_content`.
Both are ok to me, what do you think ? 🙂

Reply:
As long as the final result is good, I'm ok 🙂 
Let's go with `h-0` 👍